src/pages/projectSettings/tabs/ViewsAndFiltersTab/getFormSchema.ts

```diff
@@ -21,7 +70,30 @@ export const getFormSchema = (): ReturnType<GetFormSchema> => ({
       "ui:ObjectFieldTemplate": CardFieldTemplate,
     },
     parsleyFilters: {
-      "ui:ObjectFieldTemplate": CardFieldTemplate,
+      "ui:addButtonText": "Add new filter",
```

Comment:
[nit] For consistency with other buttons/I think the "new" is implicit
```suggestion
      "ui:addButtonText": "Add filter",
```